add-on/src/lib/ipfs-companion.js

```diff
@@ -525,14 +511,17 @@ export default async function init () {
     let iconDefinition = { path: iconPath }
     try {
       // Try SVG first -- Firefox supports it natively
-      await browser.browserAction.setIcon(iconDefinition)
+      await browser.action.setIcon(iconDefinition)
+      if (browser.runtime.lastError.message === 'Icon invalid.') {
+        throw new Error('Icon invalid.')
```

Comment:
can we just do 
```suggestion
      if (browser.runtime.lastError.message === 'Icon invalid.') {
        throw browser.runtime.lastError 
```
?